vendor/assets/iD/iD.css.erb

```diff
@@ -774,6 +789,10 @@ path.fill.tag-power-plant {
 path.stroke.tag-natural-wetland {
     stroke: rgb(153, 225, 170);
 }
+path.fill.tag-natural-wetland {
+    stroke: rgb(153, 225, 170, 0.3);
+    fill: rgb(153, 225, 170, 0.3);
```

Comment:
These two lines are causing a test failure - should they be `rgba` instead?

Reply:
Thanks, I updated those lines.